tests/Unit/ValueObject/Result/RatingTest.php

```diff
@@ -0,0 +1,20 @@
+<?php
+declare(strict_types=1);
+
+
+namespace AsyncBot\Plugin\ImdbTest\Unit\ValueObject\Result;
+
+use AsyncBot\Plugin\Imdb\ValueObject\Result\Rating;
+use PHPUnit\Framework\TestCase;
+
+class RatingTest extends TestCase
+{
+    public function testConstructorAssignsCorrectValues(): void
+    {
+        $source = "Internet Movie Database";
+        $value = "7.6/10";
+        $rating = new Rating($source, $value);
+        $this->assertEquals($source, $rating->getSource());
```

Comment:
Test only a single method per test. So this class would look like:

```php
class RatingTest extends TestCase
{
    private Rating $rating;

    public function setUp(): void
    {
        $this->rating = new Rating('Internet Movie Database', '7.6/10');
    }

    public function testGetSource(): void
    {
        $this->assertSame('Internet Movie Database', $this->rating->getSource());
    }

    public function testGetValue(): void
    {
        $this->assertSame('7.6/10', $this->rating->getValue());
    }
}
```

This prevent having to set up the unit under test inside the test method and when something fails it will give better feedback as it point directly to the error (specific test method).  
This also prevents having to have extra variables in the test method as you can just hard code the expected data.

This is feedback for all test cases (and may invalidate some other feedback as these changes supersede them)